frontend/src/main/utils/dateUtils.js

```diff
@@ -6,4 +6,15 @@ const timestampToDate = (timestamp) => {
     return (date.getFullYear() + "-" + (padWithZero(date.getMonth()+1)) + "-" + padWithZero(date.getDate()));
 }
 
-export {timestampToDate, padWithZero};
\ No newline at end of file
+// Disabling stryker here because the returned value depends on the date the test is run
+// so a deterministic test is not possible
+// Stryker disable all
```

Comment:
There's a better approach to this: mock the system time:

```js
jest.useFakeTimers().setSystemTime(new Date('2020-01-01'));
```

See also:  https://stackoverflow.com/questions/29719631/how-do-i-set-a-mock-date-in-jest/63377110#63377110